app/main/browser.js

```diff
@@ -38,100 +38,128 @@ import * as instrument from '../services/instrument';
 import * as spawn from './spawn';
 import * as BW from './browser-window';
 import UserAgentClient from '../shared/user-agent-client';
+import { argParser, parseArgs } from '../shared/environment';
+
+const SQUIRREL_EVENTS = [
+  'squirrel-install',
+  'squirrel-updated',
+  'squirrel-uninstall',
+  'squirrel-obsolete',
+];
+argParser.boolean(SQUIRREL_EVENTS);
```

Comment:
Please add a comment here to the effect of 'any additional args for the main process need to be added here, before the first call to parseArgs()'